server/chainquery/bundle.js

```diff
@@ -981,6 +986,24 @@ var claimQueries = (db, table, sequelize) => ({
     });
   },
 
+  resolveClaimInChannel: async (claimName, channelId) => {
+    logger$1.debug(`Claim.resolveClaimByNames: ${claimName} in ${channelId}`);
+    return table.findAll({
```

Comment:
Should we order these?

Reply:
This is for a single claim. I need to do a lookup by channel_id. This can be more generalized in the near future.